### ISSUE TITLE
Make bStats 429 Ratelimit errors less scary looking

### ISSUE THREAD

**Issue by Bastian** [Bastian/bStats-Metrics#86]
Currently, a huge stack trace is logged if a 429 Ratelimit errors is encountered (and `logFailedRequests` is enabled).

Since 429 are expected and are nearly guaranteed to happen once after server startup the error message should be shorter and explain that it is no problem.

**Comment by isiah-lloyd** [comment]
I agree, I was adding custom chars to my plugin and got the 429 error and assumed it was something wrong with my implementation.

**Comment by ChrissW-R1** [comment]
We use a huge stack of plugins. So this "error" occurs many times in the logs. Are there any plans to fix this in the coming releases?
The current version 3.0.0 is also affected.

We getting this error for the following list of plugins:
- BookRules
- CraftBook
- DiscordSRV
- DriveBackupV2
- EasterHunt
- Essentials
- FastAsyncVoxelSniper
- FastAsyncWorldEdit
- HeadDatabase
- mcMMO
- Multiverse-Core
- Quests
- SilkSpawners
- SkinsRestorer
- VenturaCalendar
- WorldGuard
- and also for Paper itself.

**Comment by Bastian** [comment]
> We use a huge stack of plugins. So this "error" occurs many times in the logs. Are there any plans to fix this in the coming releases? The current version 3.0.0 is also affected.

Please just set `logFailedRequests` to false. It's a debug option that is not intended to be enabled the whole time.

**Comment by ChrissW-R1** [comment]
> Please just set `logFailedRequests` to false. It's a debug option that was never intended to be enabled the whole time.

Thanks. This worked for us.
